dist/angular-validation.js

```diff
@@ -270,6 +270,28 @@
     };
 
     /**
+     * Add Message Element at the initial phase
+     * Do this function when to custom your messageElement 
+     * NODE: this funtion have similar purpose with `message-id` attribute. 
+     * This function will help you add your `messageElement` automatically instead of pre-defined.
+     * @param element
+     */
+    this.addMsgElement = function(element) {
+      return element.after('<span></span>');
```

Comment:
Actually, I agree with @huei90 at the beginning. The name itself doesn't explain what the function does, at least for me. I would prefer `set` over `add`. We don't add anything to the page, however, we set the default place where messages will go. The function might be named as `setDefaultMsgElement` or something alike.